web/content/docs/benchmarks/heat-transport-bhe/3D_coaxial_deep_BHE.pandoc

```diff
@@ -0,0 +1,64 @@
+
+| Parameter                                          | Symbol            |  Value              | Unit                        |
+| -------------------------------------------------- |:------------------| -------------------:| --------------------------: |
+| Soil thermal conductivity                          | $\lambda_{s}$     | 2.5                 | $\mathrm{W m^{-1} K^{-1}}$  |
+| Soil heat capacity                                 | $(\rho c)_{s}$    | $2.4\times10^{6}$   | $\mathrm{Jm^{-3}K^{-1}}$    |
+| Length of the BHE                                  | $L$               | 50                  | $\mathrm{m}$                |
+| Diameter of the BHE                                | $D$               | 0.216               | $\mathrm{m}$                |
+| Diameter of the outer inflow pipe                  | $d_o$             | 0.16626             | $\mathrm{m}$                |
+| Wall thickness of the outer inflow pipe            | $b_o$             | 0.00587             | $\mathrm{m}$                |
+| Wall thermal conductivity of the outer inflow pipe | $\lambda_{o}$     | 1.3                 | $\mathrm{m}$                |
+| Diameter of the inner outflow pipe                 | $d_i$             | 0.09532             | $\mathrm{m}$                |
+| Wall thickness of the inner outflow pipe           | $b_i$             | 0.00734             | $\mathrm{m}$                |
+| Wall thermal conductivity of the inner outflow pipe| $\lambda_{i}$     | 1.3                 | $\mathrm{m}$                |
+| Grout thermal conductivity                         | $\lambda_{g}$     | 0.73                | $\mathrm{W m^{-1} K^{-1}}$  |
+| Grout heat capacity                                | $(\rho c)_{g}$    | $3.8\times10^{6}$   | $\mathrm{Jm^{-3}K^{-1}}$    |
+
+
+{{< img src="../3D_coaxial_deep_BHE_figures/numerical_geometry_model.png" width="150">}}
+
+Figure 2: Numerical geometry model of the DBHE (half)
+
+## Results
+
+The numerical outflow temperature over operational time of OGS-6 was compared against OGS-5 and the analytical solution and shown in the Figure 3. And the distributed temperature of circulating water in the DBHE after operation for 3000 s was presented in the Figure 4. The comparison figures demonstrate that the numerical results and analytical results can match very well and the biggest absolute error of outflow temperature is around 1.6 $^{\circ}$C at the starting up stage, while such error will decrease to around 0.5 $^{\circ}$C after 30 days' operation. It reveals that such error will not accumulate in long term simulation. The maximum relative error regarding temperature distribution of circulating water after operation for 3000 s is around 2 \%. The soil temperature verification can be seen in the Benchmark of 3D Beier sandbox.
+
+{{< img src="../3D_coaxial_deep_BHE_figures/outflow_temperature_over_time_long-term.png" width="200">}}
```

Comment:
The results seems to fit the analytical solution. I would suggest to plot also the difference.